src/index.html

```diff
@@ -11,7 +11,207 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
```

Comment:
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
    >
```
check your code style ( [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md))
<img width="897" alt="image" src="https://user-images.githubusercontent.com/102411130/212264300-15749680-c7fc-495a-b721-676a24578f0b.png">

Reply:
fix all such cases